jb/itch5/compute_book.hpp

```diff
@@ -478,6 +496,9 @@ class compute_book {
 
   /// The live orders indexed by the "order reference number"
   orders_by_id orders_;
+
+  /// pointer to the order book config
+  typename order_book<book_type>::config const& cfg_;
```

Comment:
Please consider adding a:

    using book_type_config = typename order_book<book_type>::config;

to the class, but it if looks ugly to you, ignore this comment.